packages/commonwealth/client/scripts/helpers/feature-flags.ts

```diff
@@ -3,4 +3,6 @@ export const featureFlags = {
   communityHomepage: process.env.FLAG_COMMUNITY_HOMEPAGE === 'true',
   sidebarToggle: process.env.FLAG_SIDEBAR_TOGGLE === 'true',
   newAdminOnboardingEnabled: process.env.FLAG_NEW_ADMIN_ONBOARDING === 'true',
+  // TODO: use the proper community stakes feature flag here
+  isCommunityStakesEnabled: false,
```

Comment:
For visibility, not sure if we have an existing flag for community stakes. For now i added this placeholder.

cc: @masvelio

Reply:
Thanks, i was waiting for that to merge, will update here once merged.